SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestReplicatedStartupActor/ReplicatedActor.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "ReplicatedActor.h"
+#include "Components/StaticMeshComponent.h"
+#include "Materials/Material.h"
+
+AReplicatedActor::AReplicatedActor()
```

Comment:
Can't this use the base one from the other PR?

Reply:
It can indeed, but as far as my GIT skills go, I think I must wait for that to be merged and then pull and change this, right?

Reply:
Will probably be easier